src/modules/land_detector/land_detector_params_fw.c

```diff
@@ -102,3 +102,14 @@ PARAM_DEFINE_FLOAT(LNDFW_AIRSPD_MAX, 6.00f);
  * @group Land Detector
  */
 PARAM_DEFINE_FLOAT(LNDFW_TRIG_TIME, 2.f);
+
+/**
+ * Fixed-wing land detector: max rotation
+ *
+ * Maximum allowed angular velocity around each axis allowed in the landed state.
```

Comment:
Isn't it the norm of the rotational vector and not "around each axis"? The description of LNDMC_ROT_MAX would then also be wrong.